docs/en/sql-reference/transactions.md

```diff
@@ -3,23 +3,44 @@ slug: /en/guides/developer/transactional
 ---
 # Transactional (ACID) support
 
-INSERT into one partition* in one table* of MergeTree* family up to max_insert_block_size rows* is transactional (ACID):
-- Atomic: INSERT is succeeded or rejected as a whole: if confirmation is sent to the client, all rows INSERTed; if error is sent to the client, no rows INSERTed.
+## Case 1: INSERT into one partition, of one table, of the MergeTree* family
+
+This is transactional (ACID) if the number of rows inserted is less than or equal to `max_insert_block_size rows`, and in the case of data in TSV, TKSV, CSV, or JSONEachRow format if the number of bytes is less than `min_chunk_bytes_for_parallel_parsing`:
```

Comment:
1) Not only `TSV, TKSV, CSV, or JSONEachRow` supports parallel parsing and setting `min_chunk_bytes_for_parallel_parsing`
2) `min_chunk_bytes_for_parallel_parsing` works only when parallel parsing is enabled
3) Setting `min_chunk_bytes_for_parallel_parsing` doesn't override setting `max_insert_block_size`. It tells parallel parser the size of chunk of raw data to parse in one thread. And after chunk is parsed, it's splitted into blocks using setting `max_insert_block_size`. So, I don't think we should mention this setting here at all